app/controllers/concerns/phone_confirmation.rb

```diff
@@ -1,20 +1,20 @@
 module PhoneConfirmation
-  def prompt_to_confirm_phone(phone:, context: 'confirmation')
+  def prompt_to_confirm_phone(phone:, context: 'confirmation', prefered_delivery_method: nil)
```

Comment:
`prefered_delivery_method` seems a little weird to me in that it's different from the `current_user.otp_delivery_preference` ?

Maybe this should be `selected_delivery_method` or `chosen_delivery_method` ?